deployments/optimism.diamond.staging.json

```
@@ -49,13 +49,99 @@
+        "Name": "PioneerFacet",
+        "Version": "1.0.0"
+      },
+      "0xfEeCe7B3e68B9cBeADB60598973704a776ac3ca1": {
+        "Name": "GasZipFacet",
+        "Version": "2.0.4"
+      },
+      "0x91559A75bd9045681265C77922b3cAeDB3D5120d": {
+        "Name": "AcrossFacetV4",
+        "Version": "1.0.0"
+      },
+      "0xf536ed5A4310455FF39dBf90336e17d11550E7b4": {
```

Biome (2.1.2)

[error] The key 0xf536ed5A4310455FF39dBf90336e17d11550E7b4 was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)